doc/install_doc_component.html

```diff
@@ -118,8 +118,9 @@
       , <strong>scipy</strong>, <strong>scikit-learn</strong> (sometimes called <cite>sklearn</cite>,
           or <cite>python-sklearn</cite>), <strong>joblib</strong>,
           <strong>matplotlib</strong> (sometimes
-      called <cite>python-matplotlib</cite>) and <strong>nibabel</strong>
-      (sometimes called <cite>python-nibabel</cite>).</p>
+      called <cite>python-matplotlib</cite>), <strong>nibabel</strong>
+      (sometimes called <cite>python-nibabel</cite>) and <strong>pandas</strong>
+      (sometimes called <cite>python-pandas</cite>).</p>
```

Comment:
These look like changes from another PR. I probably have a mangled history.

Are you working from a branch, and not master? To contribute to software, you should always make sure to work in branches, in order to avoid mixing your changes.

Reply:
I was working on nistats_plot_event branch, but I added these changes in this PR since in #2468 others told me to do not open another PR, in order to save CI runs. 
So I decided to add these fix to this branch and this PR.
I don't know if something occurred when I rebased master to this branch or something else.

Reply:
Sounds good.

I'll merge once the suggestions are addressed.